src/settings/base.py

```diff
@@ -14,6 +14,7 @@
 # =============================================================================
 ALLOWED_HOSTS = ['*']
 USE_X_FORWARDED_HOST = True
+VALID_REDIRECT_DOMAINS = os.environ.get('VALID_REDIRECT_DOMAINS', ['localhost:8888'])
```

Comment:
Good catch. Interestingly this actually works the way it is currently (though, in a much less ideal manner) because of the way this 'list' is checked, using `if next_url in VALID_REDIRECT_DOMAINS`, demonstrated here: 
![image](https://user-images.githubusercontent.com/37089863/66085321-b8e3d380-e525-11e9-8618-6e7b65f9e57e.png)
Kinda quirky. Definitely changing this to use `.split()` so we can do this right.

Reply:
holy yikes...! yeah shocking that works lol, cool